frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-internal/status-card/utils.ts

```diff
@@ -31,7 +35,24 @@ const CephHealthStatus = (status: string, t: TFunction): SubsystemHealth => {
 
 export const getCephHealthState: ResourceHealthHandler<WatchCephResource> = ({ ceph }, t) => {
   const { data, loaded, loadError } = ceph;
-  const status = data?.[0]?.status?.ceph?.health;
+
+  const cephObj = data?.[0]?.status?.ceph;
+  const status = cephObj?.health;
+  const details = cephObj?.details;
+
+  const getTroubleshootLink = (key: string) =>
+    whitelistedHealthChecks.has(key) ? whitelistedHealthChecksRef[key] : null;
```

Comment:
We dont need a funstion for this, we can create an object.
```suggestion
const TroubleshootLinks = {
[HelathCheckType.MON_DOWN]: "https://www.wxample.com"
}
```
Then access as
```
TroubleshootLink[HealthCheck.MON_DOWN]
```